docs/en/observability/synthetics-migrate-integration.asciidoc

```diff
@@ -0,0 +1,167 @@
+   . Wrote journeys in JavaScript or TypeScript files.
+   . Zipped the entire project.
+   . Configured and created the monitor in the
+     Integrations UI by adding a ZIP URL that pointed to
+     the location of the project.
+a| . Create a project that uses `@elastic/synthetics`.
+   . Configure lightweight monitors in YAML files.
+   . Write journeys in JavaScript or TypeScript files and configure
+     individual monitors in your journey code using `monitor.use` or
+     configure all monitors using the `synthetics.config.ts` file.
+   . Use the `elastic/synthetics push` command to create monitors.
+
+Find more details in <<synthetics-get-started-project>>.
+
+| *How to use the UI*
+(<<synthetics-migrate-integration-ui,read{nbsp}more>>)
+a| . Went to the *Integrations* page in {kib}.
+   . Searched for and added the *Elastic Synthetics* integration.
+   . Configured the monitor.
+   . Created the monitor.
+a| . Go to the *{uptime-app}* in {kib}.
+   . Go to *Monitor Management*.
+   . Click *Add monitor*.
+   . Configure the monitor.
+   . Create the monitor.
+
+Find more details in <<synthetics-get-started-ui>>.
+|===
+
+[discrete]
+[[synthetics-migrate-integration-location]]
+= Where monitors run
+
+If you want to continue hosting on your infrastructure, you will need to create a
+private location before creating monitors.
+
+// Is this true?
+If you have already have an {agent} running using `elastic-agent-complete`,
+you can <<synthetics-private-location-add,add it as a new private location>>
+in the {uptime-app}.
+
+To create a new private location from scratch, follow all instructions in
+<<synthetics-private-location>>.
+
+Alternatively, you can start hosting on Elastic's global managed infrastructure.
+With Elastic's global managed testing infrastructure, you can create and run monitors in multiple
+locations without having to manage your own infrastructure.
+Elastic takes care of software updates and capacity planning for you.
+
+[discrete]
+[[synthetics-migrate-integration-projects]]
+= How to use projects
+
+If you already have an external project you were adding via a ZIP URL
+you can use the same project, but you will have to make some changes.
+
+First, upgrade the existing project to use the latest version of 
+`@elastic/synthetics`:
+
+. Run `npm install -g @elastic/synthetics@latest` to install
+  the latest version of the CLI.
+. Upgrade your existing project to use new project settings:
+.. Run `npm @elastic/synthetics init <path-to-existing-project>`.
+.. Respond to all prompts provided by the CLI.
+. Review updated files and directories, including:
+.. `journeys/` will contain sample journey code.
+   Move existing journey files into this directory.
+.. `synthetics.config.ts` will contain updated default settings
+   needed for the upgraded project.
+.. `package.json` will contain updated NPM settings for your project.
+.. `.github/` will contain sample workflow files to use with GitHub Actions.
```

Comment:
**Option 2: Instruct users to create a fresh project instead of "upgrading"**

```suggestion
. Run `npm install -g @elastic/synthetics@latest` to install
  the latest version of the CLI.
. Create a new project:
.. Run `npm @elastic/synthetics init <new-directory>`.
.. Respond to all prompts provided by the CLI.
. Review the generated files and directories, including:
.. `journeys/` will contain sample journey code.
   Move existing journey files into this directory.
.. `synthetics.config.ts` will contain updated default settings
   needed for the upgraded project. Manually copy over previous configuration options that are compatible with the latest version of Elastic Synthetics.
.. `package.json` will contain updated NPM settings for your project. Manually copy over previous dependencies, scripts, and other pacakage manager configuration options.
.. `.github/` will contain sample workflow files to use with GitHub Actions.
```